src/app/auth/rules/Rules.tsx

```diff
@@ -10,25 +10,21 @@ const modalAnimationDutation = 150;
 export const Rules = () => {
   const [isModalShow, setIsModalShow] = useState(false);
 
-  const handleModalOpen = () => {
-    setIsModalShow(true);
-  };
-
-  const handleModalClose = () => {
-    setIsModalShow(false);
+  const onToggleModal = () => {
```

Comment:
You could get rid of onToggleModal at all
Just use `onClick={() => setIsModal(true/false)}`